src/EditorFeatures/CSharpTest/CodeActions/IntroduceVariable/IntroduceVariableTests.cs

```diff
@@ -4628,17 +4628,33 @@ void M(int x)
         ref int y = ref x;
         M2([|(y = ref x)|]);
     }
-    void M2(ref int y) { }
+    void M2(int p) { }
 }", @"
 class C
 {
     void M(int x)
     {
         ref int y = ref x;
-        int {|Rename:v|} = (y = ref x);
-        M2(v);
+        int {|Rename:p|} = (y = ref x);
+        M2(p);
     }
-    void M2(ref int y) { }
+    void M2(int p) { }
+}");
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsIntroduceVariable)]
+        public async Task TestIntroduceLocalInRefCallRefExpression()
+        {
+            // Cannot extract expressions passed by ref
```

Comment:
Might want to comment that this is more a bug than anything else, but this at least correctly asserts we don't crash and die.